Fix regression for single cluster deployments

hoegaarden opened the pull request:
Deployment succeeds with single cluster arch (`taskcat test run`).
Multi cluster arch not tested.

satya-dillikar reviewed (commented):
all tap-setup-scripts do not use 'single' suffix for cluster-name.

look at TAP_CLUSTER_NAME in  tap-setup-scripts/src/tap-main.sh.

I am not sure why we need this fix.

hoegaarden commented:
> all tap-setup-scripts do not use 'single' suffix for cluster-name.
> look at TAP_CLUSTER_NAME in tap-setup-scripts/src/tap-main.sh.

:+1:
I misread the tap-scripts yesterday, and didn't really check the system once the sstack deployed fine, so I missed that.

(sidenote: we should eventually look into using `cfn-signal` to bubble up issues in our setup script(s); but that's for another day) 

> I am not sure why we need this fix.

[0]: https://github.com/vmware-tap-on-public-cloud/quickstart-vmware-tanzu-application-platform/blob/622520a1a4cadfd6ce52e37f999f473086dcc903/templates/aws-tap-entrypoint-existing-vpc.template.yaml#L1044
[1]: https://github.com/vmware-tap-on-public-cloud/quickstart-vmware-tanzu-application-platform/blob/622520a1a4cadfd6ce52e37f999f473086dcc903/templates/aws-tap-entrypoint-existing-vpc.template.yaml#L907

We need that, because:
- [here][0] we create the cluster and name it `${EKSClusterName}-single`
- but for the `SINGLEEBSSCIAddon` we [configure the cluster name with `${EKSClusterName}`][1]
- this eventually leads to `SINGLEEBSSCIAddon` failing with a 404, because the referenced cluster is not there, only the one with the `-single` postfix is there

So I switched it around now: changed the single cluster name back from `${EKSClusterName}-single` to `${EKSClusterName}`, so that its name aligns with other resources & scripts. I currently run that trough a `taskcat test` run and will update the PR once that succeeds.

hoegaarden commented:
> [...] and will update the PR once that succeeds.

Because TanzuNet currently is down, I can't test it e2e; however, because the stack deployed fine, both single and multi, and only failed when pulling things from TanzuNet, [I still pushed the update][p].

[p]: https://github.com/vmware-tap-on-public-cloud/quickstart-vmware-tanzu-application-platform/compare/b2d9ed854d3deeb1d216eb4497644c07ff8ab3cd..288695fd72c9ea0390ace4731277bc88ba370ab5
